server/models/dbModel.py

```diff
@@ -0,0 +1,62 @@
+from sqlalchemy.orm import relationship
+from sqlalchemy import Column, Integer, String, ForeignKey, Date, DateTime
+from ..dbConnect import Base
+
+
+class Patient(Base):
+    __tablename__ = "patient"
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    email = Column(String, unique=True)
+    name = Column(String)
+    lastname = Column(String)
+    dateOfBirth = Column(Date)
+    appointments = relationship("Appointment", back_populates="patient")
+    sensor_data = relationship("SensorData", back_populates="patient")
+    permissions = relationship("DataPermissions", back_populates="patient")
+    user_tracker = relationship("UserTracker", back_populates="patient")
+
+
+class Appointment(Base):
+    __tablename__ = "appointment"
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    DateTime = Column(DateTime)
+    Duration = Column(Integer)
+    patient_id = Column(Integer, ForeignKey("patient.id"))
+    patient = relationship("Patient", back_populates="appointments")
+    doctor_id = Column(Integer, ForeignKey("doctor.id"))
+    doctor = relationship("Doctor", back_populates="appointments")
+
+
+class Doctor(Base):
+    __tablename__ = "doctor"
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    email = Column(String, unique=True)
+    name = Column(String)
+    lastname = Column(String)
+    appointments = relationship("Appointment", back_populates="doctor")
+    permissions = relationship("DataPermissions", back_populates="doctor")
+
+
+class SensorData(Base):
```

Comment:
Maybe change name from `SensorData` to `MeasuredData`? We don't have any sensors, so it's strange to mention them